doc/make.bat

```diff
@@ -28,12 +28,32 @@ if errorlevel 9009 (
 )
 
 if "%1" == "" goto help
+if "%1" == "html-noplot" goto html-noplot
+if "%1" == "show" goto show
 
 %SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
+if "%1" == "clean" (
+	REM workaround because sphinx does not completely clean up (#11139)
+	rmdir /s /q "%SOURCEDIR%\build"
+	rmdir /s /q "%SOURCEDIR%\api\_as_gen"
+	rmdir /s /q "%SOURCEDIR%\gallery"
+	rmdir /s /q "%SOURCEDIR%\plot_types"
+	rmdir /s /q "%SOURCEDIR%\tutorials"
+	rmdir /s /q "%SOURCEDIR%\savefig"
+	rmdir /s /q "%SOURCEDIR%\sphinxext\__pycache__"
+)
 goto end
 
 :help
 %SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
+goto end
+
+:html-noplot
+%SPHINXBUILD% -M html %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O% -D plot_gallery=0
+goto end
+
+:show
+python -m webbrowser -t "%~dp0\build\html\index.html"
```

Comment:
Not sure if this or `start` is preferred? 

https://stackoverflow.com/questions/26223936/open-a-web-page-in-a-windows-batch-file